src/main/java/gmail/taras/shevchuk0024/lesson40_Functional_Interface/Consumer.java

```diff
@@ -0,0 +1,8 @@
+package gmail.taras.shevchuk0024.lesson40_Functional_Interface;
+
+public class Consumer {
+    public static java.util.function.Consumer<Integer> giveRandomNumber = maxAmount -> {
+        int result = (int) (Math.random() * (maxAmount + 1));
+        System.out.println("Випадкове число: " + result);
+    };
+}
```

Comment:
Зробіть просто `Consumer<Integer>` а "хвіст" потрібно імпортувати. У Вас основна проблема що ви клас теж назвали `Consumer` і через це джава не може розібратися де у Вас клас а де функціональний інтерфейс. Це є поганою практикою і краще клас назвати якось інакше i.e. `ConsumerUtils`.

```suggestion
package gmail.taras.shevchuk0024.lesson40_Functional_Interface;

import java.util.function.Consumer;

public class ConsumerUtils {
    public static Consumer<Integer> giveRandomNumber = maxNumber -> {
        int result = (int) (Math.random() * (maxNumber + 1));
        System.out.println("Випадкове число: " + result);
    };
}
```